pkg/frontend/init_db.go

```diff
@@ -597,6 +598,38 @@ func PrepareInitialDataForMoUser() [][]string {
 	return data
 }
 
+// DefineSchemaForMoUser decides the schema of the mo_table
+func DefineSchemaForMoIincrement() *CatalogSchema {
+	/*
+		mo_increment_columns schema
+		| Attribute |     Type     | Primary Key |             Note         |
+		| -------   | ------------ | ----------- | ------------------------ |
+		|   name    | varchar(770) |             | Name of the db_table_col |
+		|   num     |    int64     |             |   current index number   |
```

Comment:
That means every auto_increment column of every table of every database will have a single row counter. In some situation, the table `mo_increment_columns` will be the bottleneck.

Reply:
Which is probably fine.   But we have to use the "out of band transaction" semantics.